src/js/client/apply.js

```diff
@@ -77,7 +77,7 @@ function disallowAmbiguousAnswersProactively($form) {
     }
   });
 
-  const $teamFields = $form.find('input[name=team_apply], input[name=team_placement]');
+  const $teamFields = $form.find('input[id=team_team_apply], input[id=team_team_placement]');
```

Comment:
This needs to be tested by the reviewer as I don't know how to clear my local database of applications and therefore can't access the application form on my local machine

Reply:
Would recommend downloading [Postico](https://eggerapps.at/postico/).